frontend/src/app/lib/addon.js

```diff
@@ -128,9 +128,7 @@ function messageReceived(store, evt) {
     case 'sync-installed-result':
       store.dispatch(addonActions.setClientUuid(data.clientUUID));
       store.dispatch(addonActions.setInstalled(data.installed));
-      break;
-    case 'sync-installed-addons':
-      store.dispatch(addonActions.setInstalledAddons(data.installedAddons));
+      store.dispatch(addonActions.setInstalledAddons(data.active));
```

Comment:
Things get tricky here. We probably need to keep *both* these messages - or at least fail gracefully - for at least the next add-on upgrade cycle or two. That is, until/unless we require users to have the latest add-on version (#1469). Otherwise, we'll have 24 hours or so of the old add-on version hitting the site and not sending `active` in this message until all clients have done their updates.